.scalafmt

```diff
@@ -0,0 +1,51 @@
+version = 3.5.9
+runner.dialect = scala213
```

Comment:
Can we also refer HMRC other repos for patterns?
https://github.com/hmrc/play-frontend-hmrc/blob/main/.scalafmt.conf